File: src/main/java/com/multicampus/gamesungcoding/a11ymarketserver/feature/cart/dto/CartItemListResponse.java
package com.multicampus.gamesungcoding.a11ymarketserver.feature.cart.dto;

import jakarta.validation.constraints.Positive;

import java.util.List;

public record CartItemListResponse(
        List<CartItemResponse> items,

        @Positive(message = "Total must be positive")
        int total) {
}

File: src/main/kotlin/com/multicampus/gamesungcoding/a11ymarketserver/feature/cart/dto/CartItemListResponse.kt
package com.multicampus.gamesungcoding.a11ymarketserver.feature.cart.dto

import jakarta.validation.constraints.Positive

data class CartItemListResponse(
    val items: List<CartItemListDto>,

    @field:Positive(message = "Total must be positive")
    val total: Int
)